pkgs/top-level/python-packages.nix

```diff
@@ -3097,6 +3097,34 @@ in modules // {
     };
   };
 
+  mahotas = buildPythonPackage rec {
+    name = "python-mahotas-${version}";
+    version = "1.4.0";
+
+    src = pkgs.fetchurl {
+      url = "https://github.com/luispedro/mahotas/archive/release-${version}.tar.gz";
+      sha256 = "30c4b979e0d5f4c013860321766a79ffcabe56c1ad9088e5d0c6b36aec5f0415";
+    };
+
+    buildInputs = with self; [
+      nose
+      pillow
+      scipy
+    ];
+    propagatedBuildInputs = with self; [
+      numpy
+      imread
+    ];
+
+    meta = with stdenv.lib; {
+      description = "Computer vision package based on numpy";
+      homepage = https://readthedocs.org/projects/mahotas/;
+      maintainers = with maintainers; [ luispedro ];
+      license = licenses.mit;
+      platforms = platforms.linux;
```

Comment:
It's a good reason. Not sure what the protocol is actually . If I'm correct this field is generally only included when it's known *not* to work on a certain platform; the default value includes both platforms. I don't think it's too much of a problem; if a darwin user sees this, they can always change it.